hw2/04-background.html

```diff
@@ -7,23 +7,27 @@
     <link
       rel="stylesheet"
       href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      <link
+      rel="stylesheet"
+      href="styles.css">
     <script defer src="04-background.js"></script>
   </head>
   <body>
     <main>
+      <div class="container">
```

Comment:
Instead of adding a container to div, adding it to body or main could make the elements more should do.